### PR TITLE
New latex code for the ReScienceC template - Work in Progress

### PR DESCRIPTION
**Pull request by carlito387**
Remains the problem in the author's names (commas and contributions)

### CODE REVIEW

**Comment by carlito387** [comment]
#2 - linking issue

File: alacchiReUnsupervisedState/journal/metadata.yaml [hunk 1]
```diff
@@ -13,17 +13,17 @@ authors:
   - name: Gabriel Alacchi
     orcid: 
     email: gabriel.alacchi@mail.mcgill.ca
-    affiliations: 1,*
+    affiliations: 1,†,*
```
Comment (koustuvsinha) [inline alacchiReUnsupervisedState/journal/metadata.yaml]: Use `$\dagger$` instead of `†`

File: alacchiReUnsupervisedState/journal/metadata.yaml [hunk 2]
```diff
@@ -33,6 +33,8 @@ affiliations:
     name:    School of Computer Science, McGill University
     address: Montreal, Canada
 
+  - code:    †
```
Comment (koustuvsinha) [inline alacchiReUnsupervisedState/journal/metadata.yaml]: Same here, use `$\dagger$` instead of `†`

**Comment by carlito387** [comment]
@koustuvsinha The PR is updated. I have 2 more issues:
- The \subsubsection at line 30 and 41 of content.tex are not displayed properly
- We used \textbf in attempt to put bold text at lines 25, and in our tables starting at lines 84 and 205. I think this is a problem due to the font... When compiling, I get the following warnings (only the first one is important since the bullets are displaying fine):
> LaTeX Font Warning: Font shape `T1/SourceSerifPro(0)/bx/n' undefined
> (Font)              using `T1/SourceSerifPro(0)/m/n' instead on input line 1.

> LaTeX Font Warning: Font shape `OMS/SourceSerifPro(0)/m/n' undefined
> (Font)              using `OMS/cmsy/m/n' instead
> (Font)              for symbol `textbullet' on input line 69.

This [page](https://tex.stackexchange.com/questions/115869/problem-with-fonts) indicates that there is no bold on the monospaced (/m) version. I don't want to modify the official font, so I'll try to work around it tomorrow.

**Comment by rougier** [comment]
There's a bold version of Source Serif Pro (https://fonts.google.com/specimen/Source+Serif+Pro) so it might be a configuration problem in the latex template. Do you have a link to your paper ?

**Comment by carlito387** [comment]
The updated files are [in the PR](https://github.com/ReScience/NeurIPS-2019/pull/12/files), and our original paper is [here](https://github.com/ReScience/NeurIPS-2019/tree/master/alacchiReUnsupervisedState/openreview).

**Comment by koustuvsinha** [comment]
It seems to me its the latex template issue. I'm merging this PR, but it seems I made a poor design choice in the beginning to include the original template files _for each folder_ which hinders the flexibility of template update. I'll re-organize the codebase today.

**Comment by carlito387** [comment]
@koustuvsinha Do you have any update ? We still have the \textbf and \subsubsection problems.

**Pull request merged**